spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -540,6 +637,24 @@ def call_with_params(params)
         get :collected, params: { user_id: collected_user.login }
         expect(assigns(:works)).to include(work, anonymous_work)
       end
+
+      context "when filtering is disabled" do
```

Comment:
We removed this admin setting in #3546, so this can probably go.

Reply:
Fixed